composer.json

```
@@ -9,13 +9,14 @@
         }
     ],
     "require": {
```

Nedavalo by tady smysl vypnout generovani `composer.lock`?
```suggestion
    "config": {
        "lock": false
    },
    "require": {
```

---

ani nevím že taková option existuje nevím že bysme to někde používali.

---

jj, treba na Storage clientovi a par dalsich: https://github.com/search?q=org%3Akeboola+%27%22lock%22%3A+false%27+filename%3Acomposer.json&type=code

---

přidal sem